arch/mips/src/pic32mx/Kconfig

```diff
@@ -1019,6 +1019,14 @@ config PIC32MX_GPIOIRQ
 	---help---
 		Build in support for interrupts based on GPIO inputs from IOPorts
 
+config PIC32MX_JTAG_ENABLE
+	bool "Enable JTAG"
+	default y
```

Comment:
@rm5248 I suggest you to change it to "default n" to avoid the security issues that David raised.

Reply:
If you're concerned about security (I assume preventing unwanted code download capability?) then the Help for this Kconfig should document some kind of a warning to that effect.